### PR TITLE
Change from promptflow-evals to azure-ai-evaluation and add ContentSafetyEvaluator

### PR DESCRIPTION
**Pull request by slister1001**
## Purpose
Changes dependency from promptflow-evals to azure-ai-evaluation and adds ContentSafetyEvaluator to the ArticleEvaluator. 

## Does this introduce a breaking change?
[ ] Yes
[x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
```
python -m evaluate.evaluate
```

## What to Check
Verify that the following are valid
/eval_results.jsonl

### CODE REVIEW

**Review by nagkumar91 — APPROVED** [review]

File: infra/ai.yaml [hunk 1]
```diff
@@ -5,31 +5,31 @@ deployments:
     model:
```
Comment (marlenezw) [inline infra/ai.yaml:5]: To get this to work for me I had to change this file to the following 

```
# yaml-language-server: $schema=ai.yaml.json

deployments:
  - name: gpt-35-turbo
    model:
      format: OpenAI
      name: gpt-35-turbo
      version: "0613"
    sku:
      name: Standard
      capacity: 8
  - name: text-embedding-ada-002
    model:
      format: OpenAI
      name: text-embedding-ada-002
      version: "2"
    sku:
      name: "Standard"
      capacity: 8
  - name: gpt-4
    model:
      format: OpenAI
      name: gpt-4o
      version: "2024-05-13"
    sku:
      name: "GlobalStandard"
      capacity: 8
  - name: gpt-4-evals
    model:
      format: OpenAI
      name: gpt-4
      version: "turbo-2024-04-09"
    sku:
      name: "GlobalStandard"
      capacity: 8
```

File: src/api/evaluate/evaluate.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 from pathlib import Path
 from datetime import datetime
 from promptflow.core import AzureOpenAIModelConfiguration
```
Comment (marlenezw) [inline src/api/evaluate/evaluate.py:8]: We should remove this, I don't think `AzureOpenAIModelConfiguration` is being used.

**Comment by marlenezw** [comment]
I'm was getting some promptflow rate limit errors. Other than that this works as expected. I will update the PR then merge.

**Comment by marlenezw** [comment]
Worked, but took very long and had lots of rate errors. I'm trying to see why this is happening, my thought is that futures helped stop this before I think 

Screenshot 2024-10-24 at 11 48 02

**Comment by marlenezw** [comment]
Merging and will make my recommended changes in the merged branch. Mainly going to stop rate limit errors from showing in the console while I figure out if there's a way to fix them. Thank you for the PR @slister1001.

**Pull request merged**